packages/yourloops/lib/auth/models.ts

```diff
@@ -39,6 +40,9 @@ export interface AuthAPI {
   login: (username: string, password: string, traceToken: string) => Promise<Session>;
   requestPasswordReset: (username: string, traceToken: string, language?: string, info?: boolean) => Promise<boolean>;
   resetPassword: (key: string | null, username: string, password: string, traceToken: string) => Promise<boolean>;
+  signup: (username: string, password: string, role: UserRoles, traceToken: string) => Promise<Session>;
+  sendAccountValidation: (auth: Readonly<Session>, traceToken: string, language?: string) => Promise<boolean>;
```

Comment:
The trace token is already available in the session struct.

I would also have keep the session parameter name `session` instead of `auth`
```suggestion
  sendAccountValidation: (session: Readonly<Session>, language?: string) => Promise<boolean>;
```